prod.yaml.jinja

```
@@ -20,7 +20,9 @@ services:
       SMTP_SERVER: smtplocal
       {%- endif %}
     depends_on:
+      {% if postgres_version -%}
```

Apply here the suggestion for https://github.com/Tecnativa/doodba-copier-template/pull/159#discussion_r513271120, sorry.

```suggestion
      {%- if postgres_version %}
```